src/impl_bytemuck.rs

```diff
@@ -196,6 +279,38 @@ mod tests {
             value += 1;
         }
     };
+
+    /// Tests that the [`Contiguous`] impl for [`ImageFormat`] is not trivially incorrect.
+    const _: () = {
+        let mut value = 0;
+        #[expect(
+            clippy::absurd_extreme_comparisons,
+            reason = "There is only one value."
```

Comment:
`absurd_extreme_comparisons` being deny by default for both `<0` and `<=0` is unfortunate. I agree with expecting here